letsdrill.rb

```diff
@@ -1,24 +1,30 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if (integer >= 90)
+    return "A"
+  elsif (integer >= 80)
+    return "B"
+  elsif (integer >= 70)
+    return "C"
+  elsif (integer >= 60)
+    return "D"
+  else
+    return "F"
+  end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
-
+  array.sort{|a,b| a.length <=> b.length}.first
```

Comment:
This is a pretty good solution for sort. However, we can do this with a more linear time complexity if we avoid sorting algorithms. If you want an extra challenge, try to make this O(n).